service/src/main/java/uk/gov/hmcts/reform/fpl/config/GatewayConfiguration.java

```diff
@@ -0,0 +1,15 @@
+package uk.gov.hmcts.reform.fpl.config;
+
+import lombok.Data;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Configuration;
+
+@Data
+@Configuration
+public class GatewayConfiguration {
+    private final String gatewayUrl;
+
+    public GatewayConfiguration(@Value("${document_management.gateway_url}") String gatewayUrl) {
```

Comment:
Maybe gateway? And not document management, but not that fussed if you really like the doc_management prefix.